project.yaml

```
@@ -10,3 +10,12 @@ actions:
     outputs:
       highly_sensitive:
         dataset: output/dataset.csv.gz
+
+  generate_hyp_reg_dataset:
+      run: >
+        ehrql:v1
+        generate-dataset analysis/hyp_reg_dataset.py
+        --output output/hyp_reg/hyp017_viveck.csv.gz
```

let's have one subdir per clinical area
```suggestion
        --output output/hyp/hyp017_viveck.csv.gz
```